### ISSUE TITLE
Bad result, got oscillating in loss when used Focal loss

### ISSUE THREAD

**Issue by AnhPC03** [JDAI-CV/fast-reid#291]
Hello @L1aoXingyu , I wanted to use Focal loss in my model instead of mix of ```CrossEntropyLoss``` and ```TripletLoss``` as original in config file. So I has changed in **Base-Strongerbaseline.yml** and **Base-bagtricks.yml**:
```
LOSSES:
    NAME: ("TripletLoss", "CrossEntropyLoss",)
```
to
```
LOSSES:
    NAME: ("Focal Loss",)
```
Then, i had also added to the end of file **/fast-reid/fastreid/modeling/meta_arch/baseline.py** as:
```
if "Focal Loss" in loss_names:
            loss_dict['loss_focal'] = focal_loss(
                pred_features,
                gt_labels,
                self._cfg.MODEL.LOSSES.FL.ALPHA,
                self._cfg.MODEL.LOSSES.FL.GAMMA,
            ) * self._cfg.MODEL.LOSSES.FL.SCALE
```
After that, i trained my model with focal loss used pretrained weight **market_sbs_R50.pth**, config file was **sbs_R50.yml**
But when training finished, I got the loss was oscillated, and accuracy was small and weird.
Could you give me some advices or some suggestions to get high accuracy?

![image](https://user-images.githubusercontent.com/35995951/94653675-20de1d00-0326-11eb-8e57-8a4be3884aa5.png)

![image](https://user-images.githubusercontent.com/35995951/94653783-5125bb80-0326-11eb-9af1-4a277ea81f31.png)

![image](https://user-images.githubusercontent.com/35995951/94653754-42d79f80-0326-11eb-9808-f66263e9349e.png)

**Comment by L1aoXingyu** [comment]
I did not try focal loss on my own experiment, sorry for no suggestions.
